src/scripts/Providers/OSUI/Datepicker/Flatpickr/AbstractFlatpickr.ts

```diff
@@ -206,27 +206,19 @@ namespace Providers.OSUI.Datepicker.Flatpickr {
 
 			// Since native behaviour could be enabled, check if the calendar container exist!
 			if (this.provider.calendarContainer !== undefined) {
-				if (
-					this.configs.DisableMobile === true ||
-					OSFramework.OSUI.Helper.DeviceInfo.IsDesktop ||
-					this.configs.CalendarMode === OSFramework.OSUI.Patterns.DatePicker.Enum.Mode.Range
-				) {
-					// Add TodayBtn
-					if (this.configs.ShowTodayButton) {
-						this.addTodayBtn();
-					}
+				// Add TodayBtn
+				if (this.configs.ShowTodayButton) {
+					this.addTodayBtn();
```

Comment:
Shouldn't we prevent the today button logic to run when the DatePicker native is being used? (DisableMobile === true)

Reply:
Yes, but we're already... the `this.provider.calendarContainer` only exist if not native, that's why other validations can be removed ;)